openwisp_users/api/views.py

```diff
@@ -198,7 +197,7 @@ def update(self, request, *args, **kwargs):
         )
 
 
-class BaseEmailView(ProtectedAPIMixin, GenericAPIView):
+class BaseEmailView(ProtectedAPIMixin, FilterByParentOwned, GenericAPIView):
```

Comment:
I think this should be:
```suggestion
class BaseEmailView(ProtectedAPIMixin, FilterByParentManaged, GenericAPIView):
```

That allows organization managers to use the endpoint.

Reply:
Use `FilterByParentManaged` here please.